packages/amplify_datastore/test/amplify_datastore_delete_test.dart

```diff
@@ -35,37 +37,37 @@ void main() {
   });
 
 
-  test('delete returns 1 sucessful result (no query predicate)', () async {
+  test('delete with a valid model executes without an error ', () async {
+    var json = await getJsonFromFile('delete_api/request/instance_no_predicate.json');
 
-    var json = await getJsonFromFile('delete_api/response/1_deleted_result.json');
+    dataStoreChannel.setMockMethodCallHandler((MethodCall methodCall) async {});
 
-    dataStoreChannel.setMockMethodCallHandler((MethodCall methodCall) async {
-      return json;
-    });
+    Post instance = Post(title: json["model"]["title"], rating: json["model"]["rating"], created: DateTime.parse(json["model"]["created"]), id: json["model"]["id"]);
```

Comment:
Can you store the `json["model"]` part into a cached variable, so that you can DAMP-en each of these values to `model["title"]`, etc.?

Reply:
Done.